Detectors/TOF/base/src/WindowFiller.cxx

```diff
@@ -302,7 +302,7 @@ void WindowFiller::flushOutputContainer(std::vector<Digit>& digits)
       checkIfReuseFutureDigitsRO();
     }
 
-    int nwindowperTF = o2::raw::HBFUtils::Instance().getNOrbitsPerTF() * Geo::NWINDOW_IN_ORBIT;
+    int nwindowperTF = o2::tof::Utils::getNOrbitInTF() * Geo::NWINDOW_IN_ORBIT;
```

Comment:
One more information.
The reason why in MC we are no longer using HBFUtils is due to this line.
This piece of code is shared from both RawDecoding and Digitization so I need to remove the dependecy from HBFUtil here. I can re-add it in the DigitizationTask but I would do it only if it has a real impact in the simulation outputs (otherwise it is just a dependency I add for nothing)

Reply:
I checked also digitizer (inheriting from WindowFiller) and flushOutputContainer is re-defined there without any check on nwindowperTF. It means that there is no usage of this information when running MC